<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Debugging</class>
 <widget class="QWidget" name="Debugging">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbTitle">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>调试</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbPara">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>66</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>参数：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePara">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbState">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>66</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>状态：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leState">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbExit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>400</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>721</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pbSingleLight">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>点亮单个光源</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSignal">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>信      号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCurLight">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>设置超时时间</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCloseLight">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭所有光源</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_OverTime">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>113</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_OverTime">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>54</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>超时时间：</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>721</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>lbTitle</zorder>
  <zorder>lbPara</zorder>
  <zorder>lePara</zorder>
  <zorder>lbState</zorder>
  <zorder>leState</zorder>
  <zorder>pbCancle</zorder>
  <zorder>pbExit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
